src/content/zh-CN/2019/accessibility.md

```diff
@@ -0,0 +1,309 @@
+---
+part\_number: II
+chapter\_number: 9
+title: 无障碍可访问性
+description: 2019 Web Almanac 网络年鉴的无障碍易访问性章节，涵盖易阅读、媒体、易于导航和与辅助技术的兼容性。
+authors: [nektarios-paisios, obto, kleinab]
+reviewers: [ljme]
+translators: [chengxicn]
+discuss: 1764
+results: https://docs.google.com/spreadsheets/d/16JGy-ehf4taU0w4ABiKjsHGEXNDXxOlb\_\_idY8ifUtQ/
+queries: 09\_Accessibility
+published: 2019-11-11T00:00:00.000Z
+last\_updated: 2020-05-27T00:00:00.000Z
+---
+
+## 介绍
+
+互联网的无障碍可访问性对一个包容和公平的社会至关重要。随着我们的社会生活和工作生活越来越多地转移到网络世界，残疾人能够无障碍地参与所有在线互动就变得更加重要了。正如建筑设计师可以创建或省略无障碍功能，例如轮椅坡道，web开发人员可以帮助或阻碍用户依赖的辅助技术。
+
+当考虑到残疾用户时，我们应该记住他们的用户旅程通常是相同的——他们只是使用不同的工具。这些流行的工具包括但不限于:屏幕阅读器、屏幕放大器、浏览器或文本缩放以及语音控制。
+
+通常，提高站点的可访问性对每个人都有好处。虽然我们通常认为残疾人是永久性残疾，但任何人都可能有暂时的或随着环境变化的残疾。例如失明，有人可能是永久失明，有人是暂时的眼睛感染，而有人在特定的情况下失明，比如在外面强烈的阳光之下。所有这些都可以解释为什么有些人看不见自己的屏幕。每个人都有环境障碍，因此改进web页面的可访问性将改善所有用户在任何情况下的体验。
+
+[网页内容易读性指引](https://www.w3.org/WAI/WCAG21/quickref/) (WCAG) 针对如何使网页容易易阅读提供意见。这些准则被用作我们分析的基础。然而，在许多情况下，很难通过编程来分析网站的可访问性。例如，web平台提供了几种实现类似功能结果的方法，但是支持它们的底层代码可能完全不同。因此，我们的分析只是对整体网页可访问性的一个近似分析。
+
+我们将最有趣的观点分为四类:阅读的便捷性、网络媒体、页面导航的便捷性以及与辅助技术的兼容性。
+
+在测试过程中，桌面和移动设备的可访问性没有显著差异。因此除非另有说明，否则我们呈现的所有指标都是我们的桌面分析的结果。
+
+## 阅读的便捷性
+
+网页的主要目标是提供用户想要参与的内容。这些内容可能是视频或各种图像，但很多时候，它只是页面上的文本。我们的文本内容对读者来说是易读的，这是极其重要的。如果访问者不能阅读一个网页，他们就不能参与其中，最终导致他们离开。在这一节中，我们将看到网站遇到的三个方面的困难。
+
+### 色彩对比
+
+在很多情况下，访问者可能无法完美的看到你的网站。访问者可能是色盲，无法区分字体和背景颜色（欧洲血统中 [每12名男性中就有一名，每200名女性中就有一名](http://www.cvrl.org/people/stockman/pubs/1999%20Genetics%20chapter%20SSJN.pdf) ）。也许他们只是在太阳出来的时候看书，在屏幕上产生大量的眩光——严重地损害了他们的视力。又或者他们只是年纪大了，眼睛不能像以前那样区分颜色了。
+
+为了确保你的网站在这些情况下是可读的，确保你的文字与它的背景有足够的颜色对比是至关重要的。同时，要考虑当颜色转换为灰度时将显示什么对比也是非常重要的。
+
+<figure>
+  <a href="/static/images/2019/accessibility/example-of-good-and-bad-color-contrast-lookzook.svg">
+    <img alt="图1.色彩对比度不足的文本示例。由LookZook提供" aria-labelledby="fig1-caption" aria-describedby="fig1-description" src="/static/images/2019/accessibility/example-of-good-and-bad-color-contrast-lookzook.svg" width="568" height="300">
+  </a>
+  <div id="fig1-description" class="visually-hidden">四个彩色框，由棕色和灰色阴影填充，内部是白色文本。创建两个列。 左列说明颜色太浅，棕色背景色写为<code>#FCA469</code>. 右列是推荐的，棕色背景色写为<code>#BD5B0E</code>。每列的顶部框是带有棕色背景的白色文本<code>#FFFFFF</code> 底部框是带有灰色背景的白色文本 <code>#FFFFFF</code>。等效的灰度分别是 <code>#B8B8B8</code> 和 <code>#707070</code> 。由LookZook提供</div>
```

Comment:
Presume we shouldn't have a space here? Can we do a search and replace of `  。` (with space) to `。`? Also shoud you have spaces before ` <code>`?

```suggestion
  <div id="fig1-description" class="visually-hidden">四个彩色框，由棕色和灰色阴影填充，内部是白色文本。创建两个列。 左列说明颜色太浅，棕色背景色写为<code>#FCA469</code>. 右列是推荐的，棕色背景色写为<code>#BD5B0E</code>。每列的顶部框是带有棕色背景的白色文本<code>#FFFFFF</code> 底部框是带有灰色背景的白色文本 <code>#FFFFFF</code>。等效的灰度分别是 <code>#B8B8B8</code> 和 <code>#707070</code>。由LookZook提供</div>
```